kuma/static/styles/components/compat-tables/bc-table.scss

```diff
@@ -0,0 +1,263 @@
+/*
+Table layout
+- uses display: block for smaller screen sizes;
+- requires .bc-table-{$num} modifier to display columns correctly
+********************************************************************** */
+
+.text-content table.bc-table { /* specificity to over-ride .text-content styles */
+    width: 100%;
+    table-layout: fixed;
+    border: none; /* to over-ride .text-content styles */
+    border-collapse: separate; /* to over-ride .text-content styles */
+    border-spacing: 0;
+    @include rgba-fallback('background-color', $table-blue, .5);
+}
+
+.bc-table,
+.bc-table thead,
+.bc-table tbody,
+.bc-table tr,
+.bc-table th,
+.bc-table td {
+    box-sizing: border-box;
+}
+
+/* tr
+====================================================================== */
+
+.bc-table tr {
+    position: relative;
+    border-bottom: 0 solid transparent;
+}
+
+/* th & td
+====================================================================== */
+
+.bc-table th,
+.bc-table td {
+    position: relative;
+    padding: 5px 3px;
+    border-color: $bc-border-color;
+    border-top: $bc-border-width-thin solid $bc-border-color;
+    border-right: 0;
+    border-bottom: 0 solid transparent;
+    border-left: $bc-border-width-thin solid $bc-border-color;
+    text-align: center;
+}
+
+
+/* no right border on empty cell in first row */
+.bc-table thead tr:first-child td {
+    border-right: 0;
+}
+
+/* first column, fixed at 200 */
+.bc-table thead td,
+.bc-table tbody th {
+    width: 200px;
+    min-width: 200px;
+    max-width: 200px;
```

Comment:
There are two places in the PR where you need to do the same base CSS. I wonder, could we do with a `fixed-width` mixin? CSS really needs it ;)

```
@mixin fixed-width($width) {
    width: $width;
    min-width: $width;
    max-width: $width;
}
```